src/Data/Text/Internal/Fusion/CaseMapping.hs

```diff
@@ -357,8 +359,196 @@ foldMapping '\x03f0' s = Yield '\x03ba' (CC s '\x0000' '\x0000')
 foldMapping '\x03f1' s = Yield '\x03c1' (CC s '\x0000' '\x0000')
 -- GREEK LUNATE EPSILON SYMBOL
 foldMapping '\x03f5' s = Yield '\x03b5' (CC s '\x0000' '\x0000')
+-- CYRILLIC CAPITAL LETTER PE WITH DESCENDER
+foldMapping '\x0524' s = Yield '\x0525' (CC s '\x0000' '\x0000')
+-- CYRILLIC CAPITAL LETTER SHHA WITH DESCENDER
+foldMapping '\x0526' s = Yield '\x0527' (CC s '\x0000' '\x0000')
+-- CYRILLIC CAPITAL LETTER EN WITH LEFT HOOK
+foldMapping '\x0528' s = Yield '\x0529' (CC s '\x0000' '\x0000')
+-- CYRILLIC CAPITAL LETTER DZZHE
+foldMapping '\x052a' s = Yield '\x052b' (CC s '\x0000' '\x0000')
+-- CYRILLIC CAPITAL LETTER DCHE
+foldMapping '\x052c' s = Yield '\x052d' (CC s '\x0000' '\x0000')
+-- CYRILLIC CAPITAL LETTER EL WITH DESCENDER
+foldMapping '\x052e' s = Yield '\x052f' (CC s '\x0000' '\x0000')
 -- ARMENIAN SMALL LIGATURE ECH YIWN
 foldMapping '\x0587' s = Yield '\x0565' (CC s '\x0582' '\x0000')
+-- GEORGIAN CAPITAL LETTER YN
+foldMapping '\x10c7' s = Yield '\x2d27' (CC s '\x0000' '\x0000')
+-- GEORGIAN CAPITAL LETTER AEN
+foldMapping '\x10cd' s = Yield '\x2d2d' (CC s '\x0000' '\x0000')
+-- NOT FOLDED TO toLower '\x13a0'
```

Comment:
This is an example of where `toLower` doesn't work. `'\x13a0` is a `Cherokee Letter A` which case folds to itself (i.e. stays uppercase!).

This file has also plenty of added letters with known name, i.e. in `CaseFolding.txt`. Those are capital letters for which there is folding to lowercase, but `toLower` doesn't always fold them. Adding them makes `toCaseFold` behave uniformly independently of `base` version.